playbooks/roles/openvpn/defaults/main.yml

```diff
@@ -13,6 +13,9 @@ openvpn_clients:
 
 openvpn_port: "636"
 openvpn_port_udp: "8757"
+openvpn_port_sslh: "443"
 openvpn_key_size: "2048"
-openvpn_cipher: "AES-256-CBC"
+openvpn_cipher: "AES-256-GCM"
```

Comment:
We need to keep this set to something 2.3-compatible like `AES-256-CBC` in order to prevent breaking older clients. The `ncp-ciphers` option will allow 2.4 clients to opportunistically upgrade their connection to something GCM-based.